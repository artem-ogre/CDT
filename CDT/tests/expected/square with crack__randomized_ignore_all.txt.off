OFF
10 15 0
-0.8472193585307481 -0.2778174593052024 0
1.847219358530748 -0.2778174593052024 0
0.5 2.055634918610405 0
0 0 0
1 0 0
1 1 0
0.5001 1 0
0.5 0.5 0
0.4999 1 0
0 1 0
3 3 0 4
3 1 4 0
3 2 0 9
3 0 3 9
3 2 8 6
3 9 7 8
3 9 3 7
3 1 2 5
3 2 9 8
3 7 5 6
3 7 3 4
3 4 5 7
3 4 1 5
3 7 6 8
3 2 6 5
