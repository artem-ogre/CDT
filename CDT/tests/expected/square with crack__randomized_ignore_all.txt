15
0 1 3   4294967295 4 1
0 3 9   0 10 2
0 9 2   1 8 4294967295
1 2 5   4294967295 6 5
1 4 3   5 9 0
1 5 4   3 11 4
2 6 5   7 12 3
2 8 6   8 13 6
2 9 8   2 14 7
3 4 7   4 11 10
3 7 9   9 14 1
4 5 7   5 12 9
5 6 7   6 13 11
6 8 7   7 14 12
7 8 9   13 8 10

7
3 4
3 9
4 5
5 6
6 7
7 8
8 9

0

0
